lib/test/CMakeLists.txt

```
@@ -397,6 +397,7 @@ ot_check_test (FisherSnedecor_std)
 ot_check_test (FisherSnedecorFactory_std)
 ot_check_test (Frechet_std)
 ot_check_test (FrechetFactory_std)
+ot_check_test (GalambosCopula_std IGNOREOUT)
```

```suggestion
ot_check_test (GalambosCopula_std)
```
Given that the C++ test is still print-based, the output must be compared to the .expout file, otherwise the test will pass automatically!